test/rocarray/fft.jl

```diff
@@ -245,4 +245,42 @@ end
 end
 
 end
+
+@testset "Casted types" begin
```

Comment:
In what sense are these "Casted types"? It seems like we're not casting these to another type, although I'm not super familiar with the FFT code.

Reply:
You're right, they're not casted, they're promoted. E.g. Int -> Complex64, Rational -> ComplexF64, etc.  (see: https://github.com/JuliaMath/AbstractFFTs.jl/blob/master/src/definitions.jl#L205)

I'll update the PR to reflect this language.